news/_posts/2021-02-19-first-public-oss-demo.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "Announcing public demos from the maintainers team"
+subtitle: "We held the first session of the open source project's public demo"
+date:   2021-02-19 16:30:00
+authors: [ MathieuFerment, PabloBorowicz ]
+icon: icon-video-camera
+tags: [contribute, events, community]
+---
+
+Last month, the maintainers team held a special event for the open source project: a public demonstrations session. To better understand what this is about, let's provide some context.
+
+## Demonstrations at PrestaShop
+
+Historically, the maintainers team of the PrestaShop open source project was only composed of employees from the PrestaShop company, developers assigned by the company to work on the project. This team used many forms of organizations over years, the most recent being a mix of [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)) rituals and [Kanban](https://en.wikipedia.org/wiki/Kanban_(development)) method. One of the Scrum rituals being followed by the team is the [demo session](https://manifesto.co.uk/scrum-practice-sprint-demo/).
+
+Scrum's _demo_ is a regularly held event that takes place at the end of each development cycle (called _sprint_), where the team demonstrates what has been built. This helps stakeholders, inside and outside the team, to witness the outcome, what has been delivered, and provide feedback. For years, this event happened regularly, but only employees of the PrestaShop company were invited.
+
+Since 2020, the PrestaShop project is becoming increasingly independent from the PrestaShop company. One of the major milestone of his evolution has been the [introduction of community maintainers](https://build.prestashop.com/news/coreweekly-43-2020/) [@kpodemski](https://github.com/kpodemski) and [@pululuk](https://github.com/pululuk).
```

Comment:
```suggestion
Since 2020, the PrestaShop project is becoming increasingly independent from the PrestaShop company. One of the major milestone of this evolution has been the [introduction of community maintainers](https://build.prestashop.com/news/coreweekly-43-2020/) [@kpodemski](https://github.com/kpodemski) and [@pululuk](https://github.com/pululuk).
```